package.json

```diff
@@ -20,17 +20,17 @@
     "eslint": "^8.25.0"
   },
   "dependencies": {
-    "@babel/core": "^7.20.5",
+    "@babel/core": "^7.20.12",
     "@babel/eslint-parser": "^7.19.1",
     "@babel/eslint-plugin": "^7.19.1",
     "@babel/plugin-proposal-export-default-from": "^7.18.10",
-    "@typescript-eslint/eslint-plugin": "^5.44.0",
-    "@typescript-eslint/parser": "^5.44.0",
+    "@typescript-eslint/eslint-plugin": "^5.48.1",
+    "@typescript-eslint/parser": "^5.48.1",
     "babel-preset-react-app": "^10.0.1",
-    "eslint": "^8.28.0",
+    "eslint": "^8.31.0",
     "eslint-plugin-enact": "^1.0.1",
-    "eslint-plugin-jest": "^27.1.6",
-    "eslint-plugin-jsx-a11y": "^6.6.1",
+    "eslint-plugin-jest": "^27.2.1",
```

Comment:
this dependency should be left untouched in this PR since we agreed to downgrade it in another user story